src/styles.scss

```diff
@@ -28,3 +28,12 @@ body{
 .w-40 {
   width: 40% !important;
 }
+
+//Theme variables
+:root{
+  --primary-bg-color: #750C00;
+  --secondary-bg-color: #a01101;
+  --secondary-bg-hover-color: #881103;
+  --primary-txt-color: white;
+  --primary-hover-txt-color: lightgrey;
```

Comment:
Maybe it's a good idea to think about consistency. Either we use hexadecimals in the whole project or the name of a color

Reply:
true